resources.py

```diff
@@ -82,7 +84,7 @@ def get_adj_missing_mt(data_type: str, pbt: bool) -> hl.MatrixTable:
     return mt
 
 
-def _chets_out_path(data_type: str, extension: str, stage: str = '', pbt: bool = False, least_consequence: str = LEAST_CONSEQUENCE, max_freq: float = MAX_FREQ, chrom: str = None):
+def _chets_out_path(data_type: str, extension: str, stage: str = '', pbt: bool = False, least_consequence: str = LEAST_CONSEQUENCE, max_freq: float = MAX_FREQ, chrom: str = None, release: bool = False):
```

Comment:
The `release` argument looks unused.

Reply:
oops, yup, I was going to put in here but changed my mind, thank you!